bl.c

```diff
@@ -543,6 +589,8 @@ bootloader(unsigned timeout)
 	/* (re)start the timer system */
 	arch_systic_init();
 
+	//crypto_test_bench();
```

Comment:
Does this need to be #ifdefed ?

Reply:
This is leftovers from Dino's original patches, I have no need for it. I think it is ok to just remove it